mtz-url-rewrite-interceptor.html

```diff
@@ -0,0 +1,143 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="mtz-interceptor-behavior.html">
+
+<!--
+`mtz-url-rewrite-interceptor`
+An implementation of the mtz.InterceptorBehavior that injects a header:value onto a presend event for iron-ajax.
+
```

Comment:
Should this description explain more about what this specific interceptor does? I.e, that it takes the url and rewrites it to communicate with different environments?